### PR TITLE
Database and ORM Integration

### PR DESCRIPTION
**Pull request by CurranSeam**
# Summary
Previously, we were sourcing data via document store style. Here, we have added a proper database and used and an object-relational mapping approach.

## What I did here
- Added sqlite3 db
- Added peewee ORM wrapper for clean db interaction
- Added models folder for our schemas. Initial model is `User`
- Refactored service code to source from db rather than envs
- Added FE and BE code to facilitate separation of Telegram and SMS opt outs

## How to test
-

### CODE REVIEW

File: application/repository/user.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    return [user.sms_notify for user in users]
+
+def get_phone_number(chat_id=None, active=0, first_name=None):
+    if first_name and not chat_id and not active:
+        return User.get(User.first_name == first_name).phone_number
+    elif chat_id:
+        return User.get(User.telegram_chat_id == chat_id and
+                        User.sms_notify == active).phone_number
+
+def get_phone_numbers(active=False):
```
Comment (RohanSeam) [inline application/repository/user.py:38]: Maybe we should change active to be integer to stay consistent :D

**Review by RohanSeam — APPROVED** [review]
Minor things :D

File: application/repository/user.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    users = __get_users_order_by(User.id)
+
+    return [user.first_name for user in users]
+
+def get_telegram_chat_ids(active=False):
+    users = __get_users_order_by(User.id)
+
+    if active:
+        active_users = users.where(User.telegram_notify == 1)
+        users = active_users
```
[the remark this answers is not in the document]
Comment (CurranSeam) [reply application/repository/user.py]: Done :D

File: application/repository/user.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    elif chat_id:
+        return User.get(User.telegram_chat_id == chat_id and
+                        User.sms_notify == active).phone_number
+
+def get_phone_numbers(active=False):
+    users = __get_users_order_by(User.id)
+
+    if active:
+        active_nums = users.where(User.sms_notify == 1)
+        users = active_nums
```
[the remark this answers is not in the document]
Comment (CurranSeam) [reply application/repository/user.py]: Done :D

File: application/repository/user.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+def update_sms_notify(first_name, status):
+    user = User.get(User.first_name == first_name)
+    user.sms_notify = status
+
+    user.save()
+
+# Might be slightly faster to use this since it is not ordered.
+def __get_users():
+    return User.select()
```
[the remark this answers is not in the document]
Comment (CurranSeam) [reply application/repository/user.py:63]: Done :D

File: application/repository/user.py (hunk 1 above)
Comment (CurranSeam) [reply application/repository/user.py]: Done :D

**Review by RohanSeam — APPROVED** [review]

**Review by RohanSeam — APPROVED** [review]

**Pull request merged**